sdk/eventhubs/azure-messaging-eventhubs/CHANGELOG.md

```diff
@@ -4,12 +4,22 @@
 
 ### Features Added
 
+- Fully functional eventhubs Processor.
```

Comment:
nit: inconsistent capitalization

```suggestion
- Fully functional eventhubs processor.
```

Reply:
Processor is a proper noun and as such it should be capitalized.

Reply:
Really? But `eventhubs` isn't?

Reply:
That is correct. The type name for the processor component name of the eventhubs service client is "Azure::Messaging::EventHubs::Processor". The use of "Processor" as a proper name for the processor component is consistent throughout the EventHubs documentation.

"eventhubs" is a shorthand term we use to refer to the Azure EventHubs service. It's use here is directly analogous to our use of the term "keyvault" to refer to the KeyVault service.